devices/home_control_as.js

```diff
@@ -24,4 +24,27 @@ module.exports = [
         exposes: [
             e.lock(), e.battery(), e.auto_relock_time().withValueMin(0).withValueMax(3600), e.sound_volume()],
     },
+    {
+        zigbeeModel: ['HC-IWDIM-1'],
+        model: 'HC-IWDIM-1',
+        vendor: 'Heimgard Technologies',
```

Comment:
The vendor should be `Home Control AS`

Reply:
Home Control rebranded as Heimgard Technologies a while ago

Reply:
I see, renamed the file + `vendor` of other device, thanks